src/applications/find-va-forms/sass/find-va-forms.scss

```diff
@@ -0,0 +1,7 @@
+.find-va-forms-results {
+  flex-direction: column;
+
+  table {
+    margin: 0;
+  }
+}
```

Comment:
If possible, it would be cool if we could swap these our for design system utilities.